client/normal_nillable_array_of_nillables.go

```diff
@@ -158,3 +192,20 @@ func normalizeNillableCharsNillableArr[R string | []byte, T string | []byte](
 	}
 	return immutable.None[[]immutable.Option[R]]()
 }
+
+func areNormalNillableArraysOfNillablesEqual[T comparable](
+	val immutable.Option[[]immutable.Option[T]],
+	f func() (immutable.Option[[]immutable.Option[T]], bool),
+) bool {
+	if otherVal, ok := f(); ok {
+		return areNillableArraysOfNillablesEqual(val, otherVal)
+	}
+	return true
```

Comment:
suggestion: not covered in test

Reply:
nice catch, akhi! It wasn't just not covered, it misbehaved.